### ISSUE TITLE
Inexor's competitive aspect

### ISSUE THREAD

**Issue by Fohlen** [inexorgame/inexor-core#460]
I have thought about this for a long period of time now, and I will likewise want to share this with the project, rather than doing it on my own.

I want to aquire the domain enexor.org - simply stands for Inexor eSports.
The domain should provide the following services:

- you can sign up using a previously created user account, see #459 
- the server provides mix-making and ranked-games

## Ranked games
For a ranked game, I imagine the following

- ranked games are divided into leagues (one for start) and all games played in a ranked game are automatically recorded to enexor.org
  - clans should be able to sign up and determine their rooster for specific games
  - an automatic schedule will determine for the teams when to play and send the respective persons invitations
  - an ELO-based system (TrueSkill) can be used to determine scoring within ranked games
  - the system will try to automatically match people for ranked games
  - the scores can be viewed online

## Mix-making
A mix-making is basically a lobby which will wait for users to join. There is a mix-making score as well, however it will probably be not useful to do ELO-ranking here, but rather KPD. The system will try to mix people up based on their previous games (KPD). Once a lobby is ready (full) every player automatically get's invited.

## Technical aspect
In the front-end, something like a Django APP could basically determine what's happening.
On the back-end, there will be a inexor-flex instance.

- once a game starts, the front-end tells the backend to set up  a server (on a random port)
- the server will have a specific configuration, based on the type of lobby you play (e.g eCTF 3v3)
- the server will have an authentication restriction to the users that have signed up. This however can be altered if a player leaves
- optional rules should be decidable (should there be pause when someone leaves, etc)
- once the server setup is ready (which shouldn't take long) the server sends invitations to the respective players
- an invitation is simply an `inexor://` url that is broadcasted (via notifications-API)

**Comment by a-teammate** [comment]
I like the ideas, but i think this is probably out of scope for the core inexor game

Suggestion: Close this issue, but since there may arise practical issues for the main game, maybe think about what needs to be done to tailor the main game to be modifiable enough to easily allow this.

**Comment by Fohlen** [comment]
Agreed.

**Comment by Croydon** [comment]
Please use the labels wontfix/duplicate ect.
